src/pages/notePage/NotePage.tsx

```diff
@@ -125,9 +129,23 @@ const NotePage: React.FC<NotePageProps> = props => {
       };
 
       setBlocksAndSetUnsaved(blocksCopy, focusPreviousBlockEolCallback);
+    } else if (nextBlock) {
+      const blocksCopy = [...blocksRef.current];
+      const index = blocksCopy.map(b => b.id).indexOf(currentBlock.id);
+      blocksCopy.splice(index, 1);
+
+      const focusNextBlockEolCallback = () => {
+        setEol(nextBlock);
+      };
+
+      setBlocksAndSetUnsaved(blocksCopy, focusNextBlockEolCallback);
+    } else {
+      const blocksCopy = [...blocksRef.current];
+      const index = blocksCopy.map(b => b.id).indexOf(currentBlock.id);
+      blocksCopy.splice(index, 1);
+      setBlocksAndSetUnsaved(blocksCopy);
     }
```

Comment:
add a `setIsEditMode(false)` in this else block

else the click to add blocks functionality will not resume properly (requires clicking twice to add a new block as isEditMode is still true)